README.md

```diff
@@ -566,7 +592,21 @@ Forked from [AirBNB's style guide](https://github.com/airbnb/javascript)
   count();  // 3
   ```
 
-- [7.9](#7.9) <a name='7.9'></a> Never use the Function constructor to create a new function.
+  - [7.9](#7.9) <a name='7.9'></a> Always put default parameters last.
+
+    ```javascript
+    // bad
+    function handleThings(opts = {}, name) {
+      // ...
+    }
+
+    // good
+    function handleThings(name, opts = {}) {
+      // ...
+    }
+    ```
```

Comment:
Having default params last is really smart. Otherwise they are kinda pointless in certain situations. You might want to pass only of of the arguments to the function so you can use the default of the other argument. 

In the example above, having the first argument defaulted and wanting to take advantage of it would look like this
```javascript
function handleThings(opts = {}, name) {
  // ...
}

// use it
handleThings({}, 'name string');
```
That kind of defeats the purpose of the default param.

Instead, you would just have the following with defaulted params as last.
```javascript
function handleThings(name, opts = {}) {
  // ...
}

// use it
handleThings(name)
```